client/packages/coldchain/src/Sensor/ListView/ListView.tsx

```diff
@@ -102,9 +103,22 @@ export const SensorListView: FC = () => {
 
   const { isOpen, entity, onClose, onOpen } = useEditModal<SensorFragment>();
 
+  // this will open the edit modal, if the `edit` query parameter is set
+  // to a valid sensor ID. On opening, the query param is removed to
+  // prevent a loop which would happen if a sensor was edited
+  useEffect(() => {
+    const sensorId = (urlQuery['edit'] as string) ?? '';
+    if (sensorId) {
+      const sensor = data?.nodes?.find(s => s.id === sensorId);
+      if (sensor) {
+        updateQuery({ edit: '' });
```

Comment:
should we remove the edit in query even if sensor not found ?

Reply:
I guess it wouldn't hurt leaving it

Reply:
it's a matter of timing - if you remove it on load like that, then sensor id is removed before the edit modal is shown.